script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_1_1.txt

```diff
@@ -25,6 +25,6 @@ It's a little early for school, and the city is still asleep. The station is qui
 Thank you for talking to me. If you see me at school, please don't hesitate to say hello."
 <Page8>
 Waving her hands with a smile, Senpai runs off.
```

Comment:
In my opinion, "waving" already implies she's using her hands to do so. How about "Waving with a smile on her face"? Sounds more natural to me.